docker-compose.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
I think this file has the wrong permissions on it

Reply:
What permissions are supposed to be on it?